tests/integration/questionnaire/test_questionnaire_placeholders.py

```diff
@@ -50,11 +52,35 @@ def test_placeholders_rendered_in_pages(self):
         self.post({"add-item-question": "Yes"})
 
         self.assertInUrl(SUBMIT_URL_PATH)
-        self.assertInBody("Please enter the total retail turnover")
+        self.assertInBody(
+            "For Integration Testing (Integration Tests), please enter the total retail turnover"
+        )
         self.assertInBody("Please enter the value of internet sales")
         self.assertInBody("Please enter the number of items")
         self.assertInBody("Do you want to add a 3rd item?")
 
+    def test_conditional_trad_as_without_trad_as_in_token(self):
```

Comment:
I don't believe this needs a functional tests and should only be integration as it is only adjusting a string and we don't need the expense of another functional test, when this is far more appropriate. It isn't pytest just because my thinking is the whole class needs doing for one test and it would also drag in integration_test_case.py. Naturally I can do that, but the refactor would be more lines than the rest put together 😆